rasa/nlu/selectors/response_selector.py

```diff
@@ -209,7 +211,7 @@ def required_components(cls) -> List[Type[Component]]:
         RETRIEVAL_INTENT: None,
         # Boolean flag to check if actual text of the response
         # should be used as ground truth label for training the model.
-        USE_TEXT_AS_LABEL: False,
+        USE_TEXT_AS_LABEL: True,
```

Comment:
Change this back to False before merging. Set to `True` for model regression tests.